### PR TITLE
Add In progress status & CI Build may be disabled

### PR DESCRIPTION
**Pull request by nedSaf**
#166
#170
#179

### CODE REVIEW

File: client/app/scripts/controllers/repo.js [hunk 1]
```diff
@@ -13,9 +13,17 @@ angular.module('clientApp')
...
+      'queue': 'Queue',
+      'in_progress': 'In progress',
+      'error': 'Error',
+      'done': 'Done'
+    };
+
     // Separate the queue item from the list.
     angular.forEach(ciBuildItems, function(item) {
-      if (item.status != 'queue') {
+      if (item.status != 'queue' && item.status != 'in_progress') {
```
Comment (amitaibu) [inline client/app/scripts/controllers/repo.js:26]: maybe the below `$scope.ciBuildQueueItem` variable name should be changed as-well

**Comment by nedSaf** [comment]
## No Builds at all.

localhost_9000___repos_22

## No active builds

localhost_9000___repos_22

## In progress

localhost_9000___repos_22

File: client/app/scripts/controllers/repo.js (hunk 1 above)
Comment (nedSaf) [inline client/app/scripts/controllers/repo.js:26]: `ciBuildActiveItem` ?

Comment (amitaibu) [inline client/app/scripts/controllers/repo.js:26]: `ciBuildItemQueueOrInProgress`

**Comment by nedSaf** [comment]
@amitaibu Full cycle of a build:
animation

**Comment by nedSaf** [comment]
@amitaibu Ready for code review.

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.module [hunk 2]
```diff
@@ -191,12 +191,6 @@ function shoov_ci_build_create_build_item($node, $timestamp = NULL) {
  */
 function shoov_ci_build_message_update(Message $message) {
   $wrapper = entity_metadata_wrapper('message', $message);
-  $status = $wrapper->field_ci_build_status->value();
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.module:194]: this is wrong, by removing it - you might create duplicate queued items. You probably just need to move the `shoov_ci_build_trigger_event($message, 'ci_build_update');` above this, no?

**Comment by nedSaf** [comment]
It has another check but you are right, No need to delete it, Just moved the pusher trigger above it.

**Comment by nedSaf** [comment]
@amitaibu Ready for code review.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**